config/initializers/content_security_policy.rb

```diff
@@ -0,0 +1,25 @@
+# Be sure to restart your server when you modify this file.
+
+# Define an application-wide content security policy.
+# See the Securing Rails Applications Guide for more information:
+# https://guides.rubyonrails.org/security.html#content-security-policy-header
+
+# Rails.application.configure do
```

Comment:
Don't we need this?

Reply:
This is a new generated file, it wasn't here before. It'll be up to us to decide what to configure